.docker/entrypoint/lib/commander.rb

```diff
@@ -50,51 +49,60 @@ def log(severity, message)
       end
 
       def execute_args(args)
-        command = args['<command>']
+        command = find_command(args)
         verify = args['--verify']
         dry_run = args['--dry-run']
-        jekyll_config = @jekyll_config_provider.provide(command)
-        execute_command(jekyll_config, command, dry_run, verify)
-        verify(jekyll_config) if verify
+        ignore_urls = args['--ignore-url']
+        log_level = args['--log-level']
+        jekyll_config_provider = JekyllConfigProvider.new(@jekyll_env, log_level)
+        jekyll_config = jekyll_config_provider.provide(command)
+        execute_command(jekyll_config, command, dry_run, verify, log_level)
+        verify(jekyll_config, ignore_urls, log_level) if verify
       end
 
-      def execute_command(jekyll_config, command, dry_run, verify)
+      def find_command(args)
+        return 'build' if args['build'] == true
+        return 'serve' if args['serve'] == true
+        return 'deploy' if args['deploy'] == true
+
+        raise CommandLineArgumentError, 'Unkonw command'
+      end
+
+      def execute_command(jekyll_config, command, dry_run, verify, log_level)
         case command
         when 'deploy'
           deploy(jekyll_config, verify, dry_run)
-        when 'build', 'serve'
-          jekyll_command(jekyll_config, command, dry_run)
+        when 'build'
+          build(jekyll_config, dry_run, log_level)
+        when 'serve'
+          serve(jekyll_config, dry_run, log_level)
         else
           raise CommandLineArgumentError, "Unknown command '#{command}'"
         end
       end
 
-      def verify(jekyll_config)
-        verifier = provide_instance(:verify, jekyll_config)
-        verifier.verify
+      def verify(jekyll_config, ignore_urls, log_level)
+        verifier = @commands.verifier.new(jekyll_config, log_level)
+        verifier.verify(ignore_urls)
       end
 
       def deploy(jekyll_config, dry_run, verify)
-        deployer = provide_instance(:deploy, jekyll_config, @jekyll_var_dir)
+        deployer = @commands.deployer.new(jekyll_config, @jekyll_env.var_dir)
         deployer.deploy(dry_run, verify)
       end
 
-      def jekyll_command(jekyll_config, command, dry_run)
-        log(:warn, "Warning: --dry-run has no effect on the `jekyll #{command}` command.") if dry_run
+      def build(jekyll_config, dry_run, log_level)
+        log(:warn, 'Warning: --dry-run has no effect on the `jekyll build` command.') if dry_run
```

Comment:
Codacy found an issue: [Line is too long. [96/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)